dockerfiles/Dockerfile

```diff
@@ -32,6 +32,8 @@ RUN mkdir -p /fluent-bit/bin /fluent-bit/etc /fluent-bit/log
 
 ENV DEBIAN_FRONTEND noninteractive
 
+RUN echo "deb http://deb.debian.org/debian bullseye-backports main" >> /etc/apt/sources.list
```

Comment:
Let's dump this into the RUN line below just to keep layers down

Reply:
Thanks for the fast review. I merged both `RUN` statements.